### PR TITLE
Cleanup v1alpha1 Subscription codebase in EPP

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Cleanup v1alpha1 Subscription codebase in EPP
- Cleanup v1alpha1 in helm charts

**Related issue(s)**

- https://github.com/kyma-project/kyma/issues/16310

### CODE REVIEW

File: components/event-publisher-proxy/pkg/handler/handler_test.go [hunk 1]
```diff
@@ -1,33 +1,32 @@
-//nolint:lll // this test uses many long lines directly from prometheus output
+//nolint:lll // output directly from prometheus
 package handler
```
Comment (mfaizanse) [inline components/event-publisher-proxy/pkg/handler/handler_test.go:2]: Replaced this file by: https://github.com/kyma-project/kyma/blob/main/components/event-publisher-proxy/pkg/handler/handler_v1alpha2_test.go

**Review by marcobebway — APPROVED** [review]

**Pull request merged**